static/js/dashboard.js

```diff
@@ -1061,18 +1082,20 @@ function addDbSettingsEventListeners() {
 }
 
 function saveDbSetting() {
-    let trimmedDbName = $('.dbSet-dbName').val().trim();
```

Comment:
1. Don't remove this code.
2. After updating the JSON for panels, the changes are not reflected until we reload/refresh the page.

Reply:
https://github.com/user-attachments/assets/1e4887b7-d38b-4138-885b-78b1554527d8

Fixed that , kindly review @sonamgupta21